training/services/INFRASTRUCTURE.md

```diff
@@ -5,24 +5,52 @@ We use Amazon Web Services (AWS) in Digital Publishing to host all of our servic
 
 ### Pre reading
 
+#### Why cloud computing?
+>  Cloud computing is the on-demand delivery of IT resources over the Internet with pay-as-you-go pricing. Instead of buying, owning, and maintaining physical data centers and servers, you can access technology services, such as computing power, storage, and databases, on an as-needed basis from a cloud provider like Amazon Web Services (AWS). 
+> -- <cite> [What is cloud computing? - AWS](https://aws.amazon.com/what-is-cloud-computing/) </cite>
+
+This means we can:
+- acquire the resources we need as we need them, and release them when we no longer need them. 
+- experiment with new ideas quickly
+- easily scale our infrastructure (up or down) to match the demands
+
+Other cloud platforms include Google Cloud Platform and Azure, and are used by teams in the ONS. In theory, we should be able to move our infrastructure to any of the cloud platforms; in practice, this is non-trivial!
```

Comment:
Just a small suggestion: Maybe we can mention that it's non-trivial especially if we use managed services (which we do), and it is easier if you just rely on compute, memory and storage (e.g. EC2, docker, etc)? Although maybe we don't want to go into much detail (convenience vs being tied up by a cloud provider debate)